meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,130 @@
+# v0.29.0 release changelogs
+
+In this release, we integrate a really demanded feature: the possibility to custom the relevancy during the search. We also made a lot of improvements regarding the indexing speed.
```

Comment:
```suggestion
In this release, we integrate a really demanded feature: the possibility to require that all query terms are present in search results. Some users have been stuck in the pre-0.21 versions due to that. We also made a lot of improvements regarding the indexing speed.
```

I think it could be more direct and precise, it's just a suggestion.

Reply:
Integrated manually